2024-08-13 00:11:52,503 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2024-08-13 00:11:52,503 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from /Users/amatakyvi/Desktop/insta-follow-bots/venv/lib/python3.9/site-packages/instabot/bot/bot.py
2024-08-13 00:11:52,503 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2024-08-13 00:11:52,751 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [200]>
2024-08-13 00:11:52,867 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [200]>
2024-08-13 00:11:53,057 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [200]>
2024-08-13 00:11:53,203 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/contact_point_prefill/ returned response: <Response [200]>
2024-08-13 00:11:54,499 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/login/ returned response: <Response [400]>
2024-08-13 00:11:54,499 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"The password you entered is incorrect. Please try again.","status":"fail","error_title":"Incorrect password","buttons":[{"title":"OK","action":"dismiss"}],"invalid_credentials":true,"exception_name":"UserInvalidCredentials","error_type":"bad_password"}'
2024-08-13 00:11:54,499 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2024-08-13 00:11:54,499 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: The password you entered is incorrect. Please try again.
2024-08-13 00:11:54,500 - instabot version: 0.117.0 (api) - INFO - Error type: bad_password
2024-08-13 00:11:54,500 - instabot version: 0.117.0 (api) - ERROR - Failed to login go to instagram and change your password
2024-08-13 00:11:54,500 - instabot version: 0.117.0 (api) - INFO - Username or password is incorrect.
